.jenkins/sarus_run_test.sh

```diff
@@ -17,18 +18,18 @@ gcloud auth configure-docker
 pip install -r requirements.txt
 
 # Run c12 regression test on each Docker image
-declare -a tags=("hpc" "hpc-serialize")
+module load sarus
+declare -a tags=("$tagname" "${tagname}-serialize")
 for tag in ${tags}; do
```

Comment:
Could be safer to use `... in ${tags} ; do`? See issue of Jeremy doing `if [ ${tags}] ; then` and getting an error. Space matters in bash...